homework/04LessonFour.js

```diff
@@ -31,12 +31,22 @@ var numberArrayAfterUnshift = [0, 1, 2, 3, 4, 5];
 // Problem 5:
 // Use pop() and shift() to make the array bravestWarriors look like this:
 // ['Danny', 'Chris', 'Beth', 'Wallow']
-var bravestWarriors = ['Danny', 'Chris', 'Beth', 'Wallow'];
+var bravestWarriors = ['Catbug', 'Danny', 'Chris', 'Beth', 'Wallow', 'Impossibear'];
+
+bravestWarriors.pop('Impossibear');
```

Comment:
Good job answering everything! For this one, you can just do bravestWarriors.pop();

.pop() will remove the end of the array (or 'top of the stack') no matter what. .push() does need a specific thing to push.